superset-frontend/packages/superset-ui-chart-controls/package.json

```diff
@@ -35,10 +35,10 @@
     "@emotion/react": "^11.4.1",
     "@superset-ui/core": "*",
     "@testing-library/dom": "^8.20.1",
-    "@testing-library/jest-dom": "^5.11.6",
+    "@testing-library/jest-dom": "*",
     "@testing-library/react": "^12.1.5",
-    "@testing-library/react-hooks": "^5.0.3",
-    "@testing-library/user-event": "^12.7.0",
+    "@testing-library/react-hooks": "*",
+    "@testing-library/user-event": "*",
```

Comment:
Part of me is worried about the root package file breaking not-so-well-tested plugins when we do this, but the other part of me wants to make them ALL wildcards :P